flycheck-clang-analyzer.el

```diff
@@ -133,10 +133,14 @@
 ;; rtags
 (defun flycheck-clang-analyzer--rtags-active ()
   "Check if rtags is available and active."
-  (and (boundp 'rtags-enabled)
-       rtags-enabled
-       (fboundp 'rtags-is-running)
-       (rtags-is-running)))
+  (if (and (boundp 'rtags-enabled)
+           rtags-enabled
+           (fboundp 'rtags-is-running)
+           (rtags-is-running)
+           (fboundp 'rtags-compilation-flags)
+           (rtags-compilation-flags))
```

Comment:
I would prefer if instead of just `(rtags-compilation-flags)` was `(> (length (rtags-compilation-flags)) 0)` then the `(if (...) t nil)` could be removed and it could be just the `(and ...)` as original